### PR TITLE
Refactor Tags Fragment

### PR DESCRIPTION
**Pull request by xvepkj**
### Summary
- Refactored Tags Fragment

### Refactoring Naming Doc
- [Google DOC](https://docs.google.com/document/d/1LrCTI5NWcVq4as1ZT4vqMgu9WatrJqkynuMtXVTFHbM/edit?usp=sharing)

### CODE REVIEW

File: app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt [hunk 1]
```diff
@@ -84,75 +76,56 @@ class TagsFragment : Fragment() {
...
-      ResourcesCompat.getColor(resources, R.color.MagicMint, null),
-      ResourcesCompat.getColor(resources, R.color.DeepSaffron, null),
-      ResourcesCompat.getColor(resources, R.color.LightGold, null),
-      ResourcesCompat.getColor(resources, R.color.PhilippineSilver, null),
-      ResourcesCompat.getColor(resources, R.color.Deer, null),
-      ResourcesCompat.getColor(resources, R.color.Bone, null),
-      ResourcesCompat.getColor(resources, R.color.PaleRobinEggBlue, null)
-    )
+
+    val array: Array<Int> = getColors()
```
Comment (gnaagar) [inline app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt]: Use proper naming

File: app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt [hunk 2]
```diff
@@ -84,75 +76,56 @@ class TagsFragment : Fragment() {
...
-    val adapter = ArrayAdapter<Int>(
-      activity?.applicationContext!!,R.layout.spinner_main, array
-    )*/
+
     spinner.adapter = customAdapter
     // Given color should be selected
     // TODO: Fix
     var colorPos = 0
-    for (i in array.indices) {
+    for (i in array.indices)
```
Comment (gnaagar) [inline app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt]: is this necessary?

**Review by gnaagar — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt (hunk 1 above)
Comment (xvepkj) [reply app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt]: Changed to arrayColors

File: app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt (hunk 2 above)
Comment (xvepkj) [reply app/src/main/java/com/xve/scheduler/ui/tags/TagsFragment.kt]: Ahaan

**Review by gnaagar — APPROVED** [review]

**Pull request merged**